include/seqan3/alignment/matrix/detail/two_dimensional_matrix.hpp

```diff
@@ -117,7 +117,7 @@ class two_dimensional_matrix
      */
     template <std::ranges::forward_range entries_t>
     //!\cond
-        requires (!std::same_as<entries_t, storage_type>) && (std::convertible_to<value_type_t<entries_t>, value_type>)
+        requires (std::convertible_to<value_type_t<entries_t>, value_type>)
```

Comment:
```suggestion
        requires (std::convertible_to<std::ranges::value_type_t<entries_t>, value_type>)
```

Reply:
```
/seqan3/include/seqan3/alignment/matrix/detail/two_dimensional_matrix.hpp:121:52: error: ‘value_type_t’ is not a member of ‘std::ranges’
  121 |         requires (std::convertible_to<std::ranges::value_type_t<entries_t>, value_type>)
      |                                                    ^~~~~~~~~~~~
```

Need further instructions

And ranges says:

```c++
    template<typename T>
    using value_type_t RANGES_DEPRECATED(
        "ranges::value_type_t is deprecated. Please use "
        "ranges::iter_value_t instead.") = iter_value_t<T>;
```

Reply:
Hm I am confused :smile: I thought I had this specific review comment before in one of my PRs but I see now that we do not have my suggestion in the code base... sorry. I'll merge it as is.